src/lib/utils/boost/info.txt

```diff
@@ -5,5 +5,10 @@ BOOST_ASIO -> 20131228
 load_on vendor
 
 <libs>
-all -> boost_system
+linux -> rt
+mingw -> ws2_32
```

Comment:
It turned out that MinGW doesn't link without additionally adding `wsock32`. From my understanding wsock32 is a backward-compatibility wrapper around `ws2_32` though. For now, I added the include as an `--ldflags` addition in `ci_build.py`.